spark-submit.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 zip -r listenbrainz_spark.zip listenbrainz_spark/
-time ./run.sh /usr/local/spark/bin/spark-submit --master spark://spark-master.spark-network:7077 --num-executors=28 --executor-memory=1g --driver-memory=1g --py-files listenbrainz_spark.zip "$@"
-
+time ./run.sh /usr/local/spark/bin/spark-submit \
+	--packages org.apache.spark:spark-avro_2.11:2.4.1 \
+	--master spark://spark-master.spark-network:7077 \
+	--num-executors=7 \
```

Comment:
I dislike how these cluster specific things are hardcoded. That is not only a problem here but in other scripts as well. We need to start having a cluster dependent config file for these config values...